### PR TITLE
Improve some parts of the documentation.

### LINKED ISSUE #4995: [Docs] v2 Corrections / Feedback
### Do you want to request a *feature* or report a *bug*?

Bug

### Did you try using a 1.7.x configuration for the version 2.0?

- [ ] Yes
- [x] No

### What did you do?

Tried to follow or make sense of what info the docs provide.

### What did you expect to see?

Consistency, valid documented properties.

### What did you see instead?

Documentation issues/errors.

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
Version:      2.0.0-alpha6
Codename:     faisselle
Go version:   go1.12.6
Built:        2019-06-18T18:25:38Z
OS/Arch:      linux/amd64

```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

**traefik_static.toml:**

```toml
[entryPoints]
  [entryPoints.web]
    address = ":80"

[providers]
  [providers.docker]
    endpoint = "unix:///var/run/docker.sock"
  [providers.file]
    watch = true
    filename = "/config/traefik_dynamic.toml"
```

**traefik_dynamic.toml:**

```toml
# Testing if I can reference docker service, and a docker service that can reference middleware here
[providers.file]

[http]
  [http.routers]
    [http.routers.router0]
      entryPoints = ["web"]
      service = "docker.whoami_data" # appears to append `_data` where `data` is the parent directory of the compose file.
      rule = "Host(`test.localhost`)"

    [http.middlewares]
      [http.middlewares.bauth.BasicAuth]
        users = ["test:$apr1$H6uskkkW$IgXLP6ewTrSuBkTrqE8wj/"]
```

**docker-compose.yml:**

```yml
version: '3'

services:
  reverse-proxy:
    image: traefik:v2.0
    command: --api
    ports:
      - "80:80"
      - "8080:8080"
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock # So that Traefik can listen to the Docker events
      - $PWD/traefik_static.toml:/etc/traefik/traefik.toml
      - $PWD/traefik_dynamic.toml:/config/traefik_dynamic.toml
  whoami:
    image: containous/whoami
    labels:
      - "traefik.http.routers.whoami.rule=Host(`whoami.docker.localhost`)"
      - "traefik.http.routers.whoami.middlewares=file.bauth"
```

---

### Problem - Log level
[Logs](https://docs.traefik.io/v2.0/observability/logs/#log-level) has examples for each option except the log level. It lists the property and it's values, but the property is split by a space, not being familiar enough with TOML I tried with a space anyhow, as well as adopting the other convention of camel casing `logLevel` which wasn't accepted either, [1.7 docs](https://docs.traefik.io/configuration/logs/) seem to have it outside of the `[log]` section(global?) and doing this didn't raise any errors.

### Fix - Log level
Docs should be updated to remove `log ` prefix to the `level` property. Potentially with an example for good measure?

### Problem - Referencing cross-provider

[Docker provider referencing middleware from file provider](https://docs.traefik.io/v2.0/middlewares/overview/#advanced-configuration) shown as: `file@add-foo-prefix`.

When trying this I get the error(though rather than `add-foo-prefix`, mine is `bauth`, I've just swapped it here to match the doc referenced):

```
reverse-proxy_1  | time="2019-06-21T06:28:19Z" level=error msg="middleware \"docker.file@add-foo-prefix\" does not exist" entryPointName=web routerName=docker.whoami
```

Instead I need to follow the other syntax from the [March Medium article on containous blog](https://blog.containo.us/back-to-traefik-2-0-2f9aa17be305): `file.add-foo-prefix`.

### Fix - Referencing cross-provider

The `.` instead of `@` seems to be the correct way to do this.

### Typo, missing backtick

[Router rules](https://docs.traefik.io/v2.0/routing/routers/#rule) under the tip "*Combining Matchers Using Operators and Parenthesis*", OR is missing a backtick.

### Convention - casing

[`entryPoints` vs `EntryPoints`](https://docs.traefik.io/v2.0/routing/entrypoints/)(compare `Configuration Examples` section to `Configuration` snippets). I think I've seen this mixed casing around other places too. I'm not sure if it's intentional, as a new user to both Traefik and TOML, it threw me off a little as I wasn't sure if casing was important or not initially.

---

Also as feedback from a new user, I found it a bit odd when looking for information about configuration (static/dynamic info and file location) that it was only mentioned under `Getting Started` and not under the relevant configuration category. [Specifically the location for `traefik.toml`](https://docs.traefik.io/v2.0/getting-started/configuration-overview/#configuration-file) that I needed to mount in the container for traefik to discover.

Note how the section on the file provider](https://docs.traefik.io/v2.0/providers/file/) links `dynamic configuration` to the [`overview` section](https://docs.traefik.io/v2.0/providers/overview/) which doesn't really touch on dynamic configuration like [`configuration overview` does](https://docs.traefik.io/v2.0/getting-started/configuration-overview/#configuration-file). 

Perhaps the file location isn't appropriate under `Configuration Discovery` but should be relevant for the [`File` static configuration reference](https://docs.traefik.io/v2.0/reference/static-configuration/file/)?

### PR DESCRIPTION
**Pull request by ldez**
### What does this PR do?

Improve some parts of the documentation.

Fixes #4995

### Motivation

Have a good documentation.

### More

- [ ] ~Added/updated tests~
- [x] Added/updated documentation

### CODE REVIEW

File: docs/content/observability/logs.md [hunk 1]
```diff
@@ -33,13 +16,46 @@ Traefik logs concern everything that happens to Traefik itself (startup, configu
 By default, the logs are written to the standard output.
 You can configure a file path instead using the `filePath` option.
 
+```toml tab="File"
+# Writing Logs in a File
```
Comment (mpl) [inline docs/content/observability/logs.md]:
```suggestion
# Writing Logs to a File
```

File: docs/content/observability/logs.md [hunk 2]
```diff
@@ -33,13 +16,46 @@ Traefik logs concern everything that happens to Traefik itself (startup, configu
...
+[log]
+  filePath = "/path/to/traefik.log"
+```
+
+```bash tab="CLI"
+# Writing Logs in a File
```
Comment (mpl) [inline docs/content/observability/logs.md]:
```suggestion
# Writing Logs to a File
```

File: docs/content/observability/logs.md [hunk 3]
```diff
@@ -33,13 +16,46 @@ Traefik logs concern everything that happens to Traefik itself (startup, configu
...
+--log.filePath="/path/to/traefik.log"
+```
+
 #### format
 
 By default, the logs use a text format (`common`), but you can also ask for the `json` format in the `format` option.   
 
+```toml tab="File"
+# Writing Logs in a File, in JSON
```
Comment (mpl) [inline docs/content/observability/logs.md]:
```suggestion
# Writing Logs to a File, in JSON
```

File: docs/content/observability/logs.md [hunk 4]
```diff
@@ -33,13 +16,46 @@ Traefik logs concern everything that happens to Traefik itself (startup, configu
...
+[log]
+  filePath = "/path/to/log-file.log"
+  format   = "json"
+```
+
+```bash tab="CLI"
+# Writing Logs in a File, in JSON
```
Comment (mpl) [inline docs/content/observability/logs.md]:
```suggestion
# Writing Logs to a File, in JSON
```

File: docs/content/observability/logs.md [hunk 5]
```diff
@@ -5,41 +5,57 @@ Reading What's Happening
...
+--log.filePath="/path/to/traefik.log"
+--log.format="json"
+```
+
+#### `level`
+
+By default, the `level` is set to `ERROR`, but you can choose amongst `DEBUG`, `PANIC`, `FATAL`, `ERROR`, `WARN`, and `INFO`.
```
Comment (polarathene) [inline docs/content/observability/logs.md]:
```suggestion
By default, the `level` is set to `ERROR`. Alternative logging levels are `DEBUG`, `PANIC`, `FATAL`, `ERROR`, `WARN`, and `INFO`. 
```

File: docs/content/routing/entrypoints.md [hunk 6]
```diff
@@ -44,22 +44,22 @@ You can define them using a toml file, CLI arguments, or a key-value store.
 See the complete reference for the list of available options:
 
 ```toml tab="File"
-[EntryPoints]
+[entryPoints]
 
-  [EntryPoints.EntryPoint0]
+  [entryPoints.EntryPoint0]
     Address = ":8888"
     [EntryPoints.EntryPoint0.Transport]
```
Comment (polarathene) [inline docs/content/routing/entrypoints.md]:
```suggestion
    [entryPoints.EntryPoint0.Transport]
```

File: docs/content/routing/routers/index.md [hunk 7]
```diff
@@ -50,8 +52,8 @@ In the process, routers may use pieces of [middleware](../../middlewares/overvie
 
 ### EntryPoints
 
-If not specified, HTTP routers will accept requests from all defined entrypoints.
-If you want to limit the router scope to a set of entrypoints, set the entrypoints option.
+If not specified, HTTP routers will accept requests from all defined entry points.
+If you want to limit the router scope to a set of entry points, set the entry points option.
```
Comment (polarathene) [inline docs/content/routing/routers/index.md]:
```suggestion
If you want to limit the router scope to a set of entry points, set the `entryPoints` option.
```

I think the last one is meant to represent the actual option? So you shouldn't split that into two words? Backticks to further emphasize it is an option/property.

**Pull request merged**